### PR TITLE
docs: Add draft of initial research notes on existing installations processes.

### PR DESCRIPTION
**Pull request by isdampe**
This PR adds a first draft of our compiled research notes on the existing installation processes used by `skm`.

## Type of change

- [x] Documentation (update or new)

### CODE REVIEW

**Review by talzid — COMMENTED** [review]
Nice pick up @isdampe ! I have left a few suggestions for your first draft to keep it aligned with the company voice. Let me know your thoughts

File: docs/Splashkit/Operations/Distribution Channels/research-findings/existing-installation.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+[single monolithic cross-platform bash script](https://github.com/thoth-tech/skm/blob/master/install-scripts/skm-install.sh).
+This script detects operating system specific requirements, and configures itself accordingly.
+
+### Microsoft Windows
+
+Users must install the [Msys2 development environment](https://www.msys2.org/) by downloading and
+executing a GUI installer, before manually install `git` using the built-in package management
+system `pacman` before proceeding to execute the SplashKit installation script.
+
+**Potential issues**
```
[the remark this answers is not in the document]
Comment (isdampe) [reply docs/Splashkit/Operations/Distribution Channels/research-findings/existing-installation.md:17]: I'll add a semi-colon - prettier output doesn't seem to change though... 🤷‍♂️

File: docs/Splashkit/Operations/Distribution Channels/research-findings/existing-installation.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+1. Msys2 has multiple versions supporting both x86 (32 bit) and x86_64 (64 bit) architectures. It's
+   currently up to the user to determine the correct version to install, which may not be obvious
+   for less technical users.
+
+1. There's no automated testing or checking that the dependencies were installed correctly.
```
[the remark this answers is not in the document]
Comment (isdampe) [reply docs/Splashkit/Operations/Distribution Channels/research-findings/existing-installation.md:23]: Thanks @talzid 🙏

**Review by talzid — APPROVED** [review]
Looks great! Ready to merge

**Pull request merged**